### ISSUE TITLE
Change @learnpack/learnpack version to specify the 2.1.18

### ISSUE THREAD

**Issue by tommygonzaleza** [learnpack/learnpack#1113]
Go to the `.gitpod.Dockerfile` file and change the command that is installing `@learnpack/learnpack` to install it at version 2.1.18:

Like this: `npm i @learnpack/learnpack@2.1.18 -g`.

The file should look like this: 
![image](https://user-images.githubusercontent.com/56565994/185250845-66a6a94d-482b-4033-8ca6-34eb0b44f41a.png)

The arrow is showing what you should add.

Please do it on each of the LearnPack tutorials from: https://github.com/4GeeksAcademy/Interactive-Tutorials

Use this name for the branch: `learnpack-version`.

Create the PR on each repository and paste the link here

**Comment by dsilva06** [comment]
there where some repos that were already had the new learnpack version installed(master-javascript) and others that did not have a dockerfile integrated nor the learnpack package(git repo)

pr: https://github.com/breatheco-de/exercise-postcard/pull/60
* https://github.com/4GeeksAcademy/html-tutorial-exercises-course/compare/master...dsilva06:html-tutorial-exercises-course:learnpack-version
* https://github.com/4GeeksAcademy/html-forms-tutorial-exercises/pull/32
* https://github.com/4GeeksAcademy/css-tutorial-exercises-course/pull/64
* https://github.com/4GeeksAcademy/css-layouts-tutorial-exercises/pull/27
* https://github.com/4GeeksAcademy/bootstrap-exercises-tutorial/pull/39
* https://github.com/4GeeksAcademy/javascript-dom-tutorial-exercises/pull/53
* https://github.com/4GeeksAcademy/javascript-events-tutorial-exercises/pull/19
* https://github.com/4GeeksAcademy/javascript-beginner-exercises-tutorial/pull/92
* https://github.com/4GeeksAcademy/javascript-arrays-exercises-tutorial/pull/83
* https://github.com/4GeeksAcademy/javascript-functions-exercises-tutorial/pull/70
* https://github.com/4GeeksAcademy/object-oriented-javascript-tutorial-exercises/pull/12
*  https://github.com/4GeeksAcademy/react-tutorial-exercises/pull/94
* https://github.com/4GeeksAcademy/python-beginner-programming-exercises/pull/68
* https://github.com/4GeeksAcademy/python-lists-loops-programming-exercises/pull/93
* https://github.com/breatheco-de/exercise-unit-test-with-jest/pull/36
*  https://github.com/4GeeksAcademy/master-python-programming-exercises/pull/42
* https://github.com/breatheco-de/python-flask-api-tutorial/pull/49
* https://github.com/4GeeksAcademy/python-http-requests-api-tutorial-exercises/pull/14
* https://github.com/4GeeksAcademy/python-functions-programming-exercises/pull/38
hrs: <hrs>1.5</hrs>

**Comment by tommygonzaleza** [comment]
On this one: https://github.com/4GeeksAcademy/html-tutorial-exercises-course/compare/master...dsilva06:html-tutorial-exercises-course:learnpack-version

It says 2.2.1.18 instead of 2.1.18

**Comment by tommygonzaleza** [comment]
That is the only one you have to change, the rest of the PRs are fine, except for the one of javascript-beginner which has all the information from another issue that is not in the main repo yet

**Comment by dsilva06** [comment]
before: https://github.com/4GeeksAcademy/html-tutorial-exercises-course/compare/master...dsilva06:html-tutorial-exercises-course:learnpack-version 
after: https://github.com/4GeeksAcademy/html-tutorial-exercises-course/compare/master...dsilva06:html-tutorial-exercises-course:learnpack-version

**Comment by tommygonzaleza** [comment]
Hrs: <hrs>1.5</hrs>
